src/Smalot/PdfParser/Parser.php

```diff
@@ -214,6 +214,12 @@ protected function parseObject(string $id, array $structure, ?Document $document
                         // It is not necessary to store this content.
 
                         return;
+
+                    } else if ($header->get('Type')->equals('Metadata')) {
+
+                        // Attempt to parse XMP XML Metadata
+                        $document->getXMPMetadata($content);
```

Comment:
Just curios: can we be sure that `$content` is valid XML at this point?

Reply:
No, at this point it's just sending the content of the whole block as a string. The check for valid XML is in getXMPMetadata at the xml_parse_into_struct() statement. I supoose there could be a few more checks to see if this is a valid XML block of *type* XMP. It's just assuming any valid XML is an XMP block.